src/Pages/NewAccount/NewAccount.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Redirect } from "react-router";
+import styled from "styled-components";
+import GlobalStyles, {
+  MoveCenter,
+  Font,
+  Theme,
+} from "../../Styles/GlobalStyles.js";
+import EmailValCheck from "../NewAccount/EmailValCheck/EmailValCheck";
+import PasswordValCheck from "../NewAccount/PasswordValCheck/PasswordValCheck";
+
+function NewAccount() {
+  const [register, setRegister] = useState({
+    first_name: "",
+    last_name: "",
+    email: "",
+    password: "",
+  });
+
+  const handleChange = (e) => {
+    setRegister({ ...register, [e.target.name]: e.target.value });
+  };
+
+  var pwPattern = /^(?=.*[a-zA-Z])(?=.*[0-9]).{8,}$/;
+
+  const createAccount = (e) => {
+    fetch("http://18.222.175.48:8000/user/signup", {
+      method: "POST",
+      body: JSON.stringify({
+        first_name: register.first_name,
+        last_name: register.last_name,
+        email: register.email,
+        password: register.password,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        if (res.message === "SUCCESS") {
+          return <Redirect to="/login" />;
+        }
+        // console.log(res);
+        // setLogin(res);
+      });
+  };
+
+  return (
+    <NewAccountContainer>
+      <NewAccountBox>
+        <Title>NEW ACCOUNT</Title>
+        {LABEL_OBJ.map((el, idx) => {
+          return (
+            <Label key={idx}>
+              {el[0]}
+              <Input
+                type={el[1]}
+                onChange={handleChange}
+                name={el[2]}
```

Comment:
```suggestion
        {LABEL_OBJ.map((el, idx) => {
          const [foo, bar, baz] = el
          return (
            <Label key={idx}>
              {foo}
              <Input
                type={bar}
                onChange={handleChange}
                name={baz}
```
배열 구조분해 할당을 통해 각 아이템에 이름을 지어줄 수 있습니다!

Reply:
수정했습니다.